package.json

```diff
@@ -13,7 +13,10 @@
     "build": "npm run make",
     "buildAndSign": "NODE_ENV=sign npm run make",
     "publish": "electron-forge publish --arch arm64,x64",
-    "lint": "echo \"No linting configured\""
+    "lint": "echo \"No linting configured\"",
+    "vite:dev": "vite dev",
+    "vite:build": "vite build",
+    "vite:preview": "vite preview"
```

Comment:
todo: make sure the new vite:build is incorporated in the build script, and works